src/ui/selection_plugin.h

```diff
@@ -21,6 +22,7 @@ class Selection_Menu : public FishUIViewerPlugin {
 
 private:
     Eigen::RowVector4f old_viewport;
+    float scaling_factor = 2.f;
```

Comment:
Are we using this scaling factor at all? If not, let's delete it. If so, let's wrap it in an `ifdef`, rename it and add a comment.